app/controllers/events_controller.rb

```diff
@@ -38,12 +38,20 @@ def preview
    @event_reviews = @event.event_reviews
   end
 
+  def analytics
+    @event = Event.find(params[:id])
+    @songs = @event.songs
+    @event_reviews = @event.event_reviews
```

Comment:
@event_reviews is not needed as the reviews can be accessed using @event.event_reviews inside the view

Reply:
I see!!